src/bootstrap.php

```diff
@@ -580,8 +597,8 @@ function (Request $request, string $number) use ($app) {
 
         $latestVersion = count($app['articles'][$number]['versions']);
 
-        $subRequest = Request::create('/articles/'.$number.'/versions/'.$latestVersion, 'GET', array(),
-            $request->cookies->all(), array(), $request->server->all());
+        $subRequest = Request::create('/articles/'.$number.'/versions/'.$latestVersion, 'GET', [],
+            $request->cookies->all(), [], $request->server->all());
```

Comment:
This is not related but was cleaned up automatically. Will create a new PR just for this.